### DISCUSSION TITLE
[Zigbee] How to change attribute reporting frequency for an unknown cluster

### DISCUSSION THREAD

**Discussion by catalinsanda** [arendst/Tasmota#20613] [Q&A] [answered]
Hello,
I've recently bought an IKEA VINDSTYRKA air quality sensor and added it to my Zigbee network. 
I've enabled the VOC & PM2.5 reporting using `ZbBind {"Device":"IKEA_VINDSTYRKA1", "Endpoint":1, "Cluster":0xFC7E}` and `ZbBind {"Device":"ZbAir1", "Endpoint":1, "Cluster":0x042A}` and now all values are being reported.

The issue I'm having is the device is now reporting them every second, regardless if something has changed or not.

For temperature and humidity, I've managed to configure the attribute reporting frequency using a command like `ZbSend {"Device":"IKEA_VINDSTYRKA1","Config":{"Temperature":{"MinInterval":300,"MaxInterval":3600,"ReportableChange":1}}}`, but I don't know how the command looks like for unknown clusters like `0xFC7E` or `0x042A`. 

Thanks,
Catalin

**Comment by catalinsanda** [comment, marked as answer]
OK, leaving this here for folks that might need it.

The syntax is this:
`ZbSend {"Device":"IKEA_VINDSTYRKA1","Config":{"042A/0000%39":{"MinInterval":10,"MaxInterval":120,"ReportableChange":1}}}`

It was documented at https://tasmota.github.io/docs/Zigbee/#zigbee-device-plugin-format .

**Reply by Raito00** [reply]
Hello!
Nice info. 
But can You please more explain about this part: "042A/0000%39"   ----> why 0000%39

i found about %39 - AnalogInMaxValue and AnalogInMinValue but not clear about full part: 0000%39

Also use 0000%39 for cluster 0xFC7E ?  like: "FC7E/0000%39"  ?

**Reply by s-hadinger** [reply]
When configuring the reporting of an attribute, Tasmota must know the cluster, attribute number and attribute type. Hence `042A/0000%39`: cluster `0x042A`, attribute `0x0000`, type `0x39` (single precision float).

When an attribute is also known by alias, you can use it by name instead. But it's not the case for cluster `0x042A`

**Comment by catalinsanda** [comment]
The 0000 is the attribute number in cluster 0429, where 39 or 0x27 in HEX is the UInt64 Data type (more info here: https://developer.smartthings.com/docs/edge-device-drivers/zigbee/data_types.html#st.zigbee.data_types.Uint64).

0xFC7E is the cluster for TVOC right? I don't have access to the device right now, but looking at https://raw.githubusercontent.com/kkossev/Hubitat/development/Drivers/VINDSTYRKA%20Air%20Quality%20Monitor/VINDSTYRKA_Air_Quality_Monitor_lib_included.groovy it seems that yes, attribute 0x0000 is the one you're looking for.
